src/transactions/utils.ts

```diff
@@ -1,3 +1,4 @@
+import { uniqBy } from 'lodash'
```

Comment:
nit: I know we're already using lodash in a bunch of places, but maybe if we had an implementation not using it, maybe we shouldn't touch it so it's easier to eventually get rid of lodash as a whole?

Reply:
this is news to me, i thought lodash was a good lib. why do we want to move away? i would have needed to introduce more logic to this function to address Jacob's comment above. i already find this function a little hard to read, so lodash seemed like the more readable and efficient option here. do we have a list of dependencies that we want to move away from?

Reply:
oh yes this is really a side comment.
Maybe we can bring this up at the next eng team meeting? lodash is perfectly fine, but there's a bunch of things it does which are now unnecessary with modern JS.

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore

Reply:
yeah, we can discuss. anyway, i've removed lodash here in case